### PR TITLE
[FEATURE] add more free online resources to learn programming

### PR DESCRIPTION
**Pull request by alaskaa**
Added in a few more resources for the resources section to learn programming.

- Odin Project for a full free curriculum
- Codecademy for a good free introduction to many topics
- Khan Academy for Programming and Computer Science fundamentals

### CODE REVIEW

File: online-resources/learn-to-code-resources.md [hunk 1]
```diff
@@ -4,9 +4,12 @@
 
 * [Code First Girls: Teach Yourself](https://www.codefirstgirls.org.uk/teach-yourself.html)
 * [Free Code Camp](https://learn.freecodecamp.org/)
+* [The Odin Project](https://www.theodinproject.com/home)
```
Comment (charlottebrf) [inline online-resources/learn-to-code-resources.md]: Not heard of this one thanks!

File: online-resources/learn-to-code-resources.md [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
 * [Exercism.io](http://exercism.io/)
 * [Codewars](https://www.codewars.com/)
 * [Codebar Tutorials](http://tutorials.codebar.io/)
+* [Khan Academy - Computing](https://www.khanacademy.org/computing)
```
Comment (charlottebrf) [inline online-resources/learn-to-code-resources.md]: Interesting thanks, so I have both these in another link but I think this shows I should extract out all the links & add them here so it's a one stop shop & people don't have to follow links. Would you agree?

Comment (alaskaa) [reply online-resources/learn-to-code-resources.md]: Yes, I would have expected these here - it's where I'd go if I want to find more information. I leave it up to you to decide where you want to put them in the long run :)

File: online-resources/learn-to-code-resources.md (hunk 1 above)
Comment (alaskaa) [reply online-resources/learn-to-code-resources.md]: It seems like a really good and free full on curriculum site. A bit similar to freeCodeCamp.

Comment (alaskaa) [reply online-resources/learn-to-code-resources.md]: It seems like a really good and free full on curriculum site. A bit similar to freeCodeCamp.

**Pull request merged**